roles/openshift_logging_fluentd/files/logging_priority_class.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: scheduling.k8s.io/v1beta1
+kind: PriorityClass
+metadata:
+  name: cluster-logging
+value: 99999
+globalDefault: false
+description: "This priority class for the Cluster-Logging Fluentd Application"
```

Comment:
It is most important for fluentd.  The reason we need this is because if fluentd does not have high enough priority, it will be evicted from the node.  IIRC, priority was added to fix a bug that fluentd was not being scheduled on nodes or was being evicted from nodes.

Reply:
I'm suggesting we should consider moving this into the 'openshift-logging' role and not describe it to be specific to Fluentd.  Is there any reason to think fluent needs its own priority vs maybe curator or elasticsearch

Reply:
yes, there is reason - we've only had the eviction problem with fluentd, never with curator/elastic - also, fluentd is the only component that has to run on every node, not just infra nodes

Reply:
But the fact on which nodes we run is immaterial.  The priority class defines priority for pods on a node.  The fact we dont set it on ES just means that is a consideration excluded from scheduling.